api/src/main/java/io/lumeer/engine/api/constraint/DateTimeConstraintType.java

```diff
@@ -48,6 +51,38 @@
       return REGISTERED;
    }
 
+   private BiFunction<Object, Class, Object> getEncodeFunction(final DateFormat format) {
```

Comment:
I did think about it. I even considered putting it to a separate class for all constraints. But something prevented me to do so. Will have a look...